src/main/scala/z3/scala/dsl/Operands.scala

```diff
@@ -85,6 +85,139 @@ object Operands {
     }
   }
 
+  class BitVectorOperand private[dsl](val tree : Tree[_ >: BottomSort <: BVSort]) {
+
+    def unary_! : BitVectorOperand= {
+      new BitVectorOperand(BVNot(tree))
+    }
+
+    def reducedAnd : BitVectorOperand= {
+      new BitVectorOperand(BVRedAnd(tree))
+    }
+
+    def reducedOr : BitVectorOperand= {
+      new BitVectorOperand(BVRedOr(tree))
+    }
+
+    def |(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVOr(tree, other.tree))
+    }
+
+    def &(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVAnd(tree, other.tree))
+    }
+
+    def ^(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVXor(tree, other.tree))
+    }
+
+    def nand(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVNand(tree, other.tree))
+    }
+
+    def nor(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVNor(tree, other.tree))
+    }
+
+    def xnor(other : BitVectorOperand) : BitVectorOperand= {
+      new BitVectorOperand(BVXnor(tree, other.tree))
+    }
+
+    def unary_~ : BitVectorOperand= {
+      new BitVectorOperand(BVNeg(tree))
+    }
+
+    def *(other : BitVectorOperand) : BitVectorOperand = {
+      new BitVectorOperand(BVMul(tree, other.tree))
+    }
+
+    def -(other : BitVectorOperand) : BitVectorOperand = {
+      new BitVectorOperand(BVSub(tree, other.tree))
+    }
+
+    def /(other : BitVectorOperand) : BitVectorOperand = {
```

Comment:
I think people will assume that default operations (without the ~) are signed, as this would be the case in Java/Scala.